store/forms.py

```diff
@@ -46,4 +46,4 @@ class Meta:
 class BookForm(forms.ModelForm):
     class Meta:
         model = Book
-        fields = ["title", "author", "category", "price", "image", "description"]
+        fields = ["title", "author", "category", "image","price", "description"]
```

Comment:
Why did you use the list here for fields? Can we use anything else to improve memory efficiency?

Reply:
we could use __all__ to include these fields but i have one more field seller which i didn't wanted to include in the forms, but i think i could have used exclude so i had to only define seller field, am i correct?

Reply:
@MansiS117 I am just asking about what would be the best between list or tuple here? 

and we can use exclude as well as you said. but instead of list we should use tuple here because we wouldn't want to mutate fields.

Reply:
okay sir, understood